integration-cli/docker_cli_run_unix_test.go

```diff
@@ -270,7 +270,7 @@ func (s *DockerSuite) TestRunOOMExitCode(c *check.C) {
 	errChan := make(chan error)
 	go func() {
 		defer close(errChan)
-		out, exitCode, _ := dockerCmdWithError("run", "-m", "4MB", "busybox", "sh", "-c", "x=a; while true; do x=$x$x$x$x; done")
+		out, exitCode, _ := dockerCmdWithError("run", "-m", "40MB", "busybox", "sh", "-c", "x=a; while true; do x=$x$x$x$x; done")
```

Comment:
This is intentional, test fails for smaller values with GCCGO due to conversion issues, investigating to patch GCCGO. I can skip the test if you do not like this change. It may slow down the test on x86.